internal/cluster/cluster.go

```diff
@@ -24,6 +28,8 @@ type Cluster struct {
 	Cib    cib.Root    `mapstructure:"cib,omitempty"`
 	Crmmon crmmon.Root `mapstructure:"crmmon,omitempty"`
 	SBD    SBD         `mapstructure:"sbd,omitempty"`
+	Id     string      `mapstructure:"id"`
+	Name   string      `mapstructure:"name,omitempty"`
```

Comment:
Minor details
Maybe we should remove the `omitempty` from `Name` as without the name other functionalities could fail.
What do you think?

Reply:
Agreed, if Id exists then same should too.